File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        
        vector<int> result(nums.size(), 0);
        
        if(nums.empty())
            return result;
        
        int sum = 0;
        
        for( int i = 0; i < nums.size(); i++)
        {
            sum += nums[i];
            result[i] = sum;   
        }    
        
        return result;
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        
        results = []
        sum = 0
        
        for i in range(len(nums)):
            sum += nums[i]
            results.append(sum)
        
        return results